20231212-modular-models.md

```diff
@@ -103,14 +103,14 @@ When using Modular Models, developers will need to create a `fga.mod` file that
 # fga.mod
 schema: 1.2
 contents:
-  - core.fga
-  - jira/model.fga
-  - confluence/model.fga
+  - core.module.fga
```

Comment:
Can you add a note that `module` has been added to the file name for the sake of clarity in this RFC, It is not required in order to compose modular models.

Reply:
Maybe we can call the jira-module.fga to make it more obvious

Reply:
Does this work?

**Note**: Adding `module` in the file name can help with file organization to better describe the content in the file, but it is not required to compose modular models. `jira.module.fga`, `jira-module.fga`, and `jira.fga` will all work as long as the `.fga` file extension is used.